pkg/replication/replication.go

```diff
@@ -145,6 +152,9 @@ func (c *Config) AddRule(opts Options) error {
 	if c.Role == "" {
 		c.Role = arnStr
 	}
+	if opts.ReplicateSync == ReplicateSync {
+		c.ReplicateSync = true
```

Comment:
```suggestion
		c.ReplicateSync = opts.ReplicateSync == ReplicateSync
```

Reply:
your suggestion would work if this option is rule specific - it is intended at the bucket level and user can specify --sync flag on any rule and it applies to the configuration rather than at a rule prefix level.